### ISSUE TITLE
Just updated Comfy and now GGUF models are no longer recognized in the GGUF Dual or Triple Clip loader

### ISSUE THREAD

**Issue by TripleHeadedMonkey** [city96/ComfyUI-GGUF#171]
As the title says

![image](https://github.com/user-attachments/assets/dfabf696-37f5-44a0-b4b7-fc431d0ba0dc)

![image](https://github.com/user-attachments/assets/01c706b2-8206-4078-a2fb-2205c2caea68)

![image](https://github.com/user-attachments/assets/e06d3427-384c-4fdd-b269-22f4b5a15f92)

As shown in the screenshots. This began occuring after updating comfyui today. I then updated the GGUF nodes but it didn't help.

**Comment by Ratinod** [comment]
Did you recreate "GGUF Dual" or "GGUF Triple Clip loader" in the workflow? Sometimes updating nodes requires recreating them in the workflow.

**Comment by TripleHeadedMonkey** [comment]
Yeah, I did. I've managed to "fix" the issue by re-installing ComfyUI from scratch. It must have been an issue with compatibility between older versions of the UI and the latest full releases. My version of Comfy had been installed for well over 6 months and the update function stopped working correctly also I found.

So maybe not a widespread issue, but if anyone else finds similar, the solution is to start from scratch with the latest Comfy Build.
